its/ruling/src/test/resources/expected_extended/calibre/python-S930.json

```diff
@@ -23,4 +23,7 @@
 'calibre:src/calibre/library/prefs.py':[
 105,
 ],
+'calibre:src/calibre/utils/certgen.py':[
+63,
```

Comment:
Looks like a TP, the function only takes 1 parameter in both Python 2 and Python 3.